lnwire/dyn_propose.go

```diff
@@ -52,12 +52,6 @@ type DynPropose struct {
 	// re-negotiate.
 	ChanID ChannelID
 
-	// Initiator is a byte that identifies whether this message was sent as
-	// the initiator of a dynamic commitment negotiation or the responder
-	// of a dynamic commitment negotiation. bool true indicates it is the
-	// initiator
-	Initiator bool
```

Comment:
Now just implicit in protocol?

Reply:
Correct. It's implicit by who owns the quiescence leader role.